### DISCUSSION TITLE
Issue with "Table" Component and Row Selection for "Export Data" Feature

### DISCUSSION THREAD

**Discussion by walter-ribas** [Budibase/budibase#15275] [Help]
Hi everyone,

I'm having trouble with the "Table" component in Budibase after the recent changes to its functionality. Specifically, I can't figure out how to display the checkbox column that allows users to select rows using the "row select" feature.

Even after trying to configure the table, the checkbox column doesn't appear, making it impossible to select rows. When I click the Export Data button, I receive this error message: "Please select at least one row."

Without the checkbox column, I can't select rows, and the export functionality isn't usable.
Has anyone else encountered this issue or figured out how to enable the "row select" checkbox column in the updated table component? Any help or guidance would be greatly appreciated!

I'm using version 3.2.29.

Thanks in advance!

**Comment by mikesealey** [comment]
Hey @walter-ribas - this sounds like a bug. Hiding or displaying the checkbox in the table component isn't optional, they should be on display at all times unless you've got some CSS or other custom settings at play.

Can you confirm that you're on the latest version of Budibase, and that you're using the Table component, not the deprecated Table Block? If that's the case please submit a bug report, making sure to follow the bug report template, including screenshots and an app export.
